WordPress/src/main/java/org/wordpress/android/ui/mysite/SiteInfoBlockBuilder.kt

```diff
@@ -17,7 +19,8 @@ class SiteInfoBlockBuilder
         titleClick: (SiteModel) -> Unit,
         iconClick: (SiteModel) -> Unit,
         urlClick: (SiteModel) -> Unit,
-        switchSiteClick: (SiteModel) -> Unit
+        switchSiteClick: (SiteModel) -> Unit,
+        activeTask: QuickStartTask?
```

Comment:
I think passing a boolean here instead of the active task would make the signature more consistent with the other parameters. What do you think?

Reply:
I was thinking about this and I can't decide which option is better :-) but I'm ok with changing it to Boolean